examples/pylab_examples/pythonic_matplotlib.py

```diff
@@ -36,19 +36,27 @@
 
 The syntax of set is
 
+.. code::
```

Comment:
You could use slightly more lightweight markup: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#how-to-include-simple-code